spec/ace.md

```diff
@@ -23,7 +23,9 @@ This UUID is exposed to the pod through the [Metadata Service](#app-container-me
 
 #### Filesystem Setup
 
-Each app in a pod will start chrooted into its own unique read-write filesystem before execution.
+Each app in a pod will start chrooted into its own unique filesystem before execution.
+If the app's entry in the `PodManifest` has the field `readOnlyRootFS` is set to `true`,
```

Comment:
sorry, two more things I missed:
a) put these on one line (one-sentence-per-line is still the convention here)
b) s/for each app/for that app,/